### PR TITLE
Bug/mngsite 416 getting started gives build failure

### PR DESCRIPTION
**Pull request by dpd73**
https://issues.apache.org/jira/browse/MNGSITE-416

### CODE REVIEW

File: content/apt/guides/getting-started/index.apt [hunk 1]
```diff
@@ -370,40 +378,47 @@ mvn install
...
-[INFO] BUILD SUCCESSFUL
-[INFO] ----------------------------------------------------------------------------
-[INFO] Total time: 5 seconds
-[INFO] Finished at: Tue Oct 04 13:20:32 GMT-05:00 2005
-[INFO] Final Memory: 3M/8M
-[INFO] ----------------------------------------------------------------------------
+[INFO] 
+[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ my-app ---
+[INFO] Installing <dir>/my-app/target/my-app-1.0-SNAPSHOT.jar to local-repository/com/mycompany/app/my-app/1.0-SNAPSHOT/my-app-1.0-SNAPSHOT.jar
+[INFO] Installing /dir>/my-app/pom.xml to /local-repository/com/mycompany/app/my-app/1.0-SNAPSHOT/my-app-1.0-SNAPSHOT.pom
```
Comment (mthmulders) [inline content/apt/guides/getting-started/index.apt:415]: I think `/local-repository` should read `<local-repository>` in both lines.

**Comment by mthmulders** [comment]
Thanks for updating this guide! Overall it looks good to me, I have left one small remark.

File: content/apt/guides/getting-started/index.apt (hunk 1 above)
Comment (dpd73) [reply content/apt/guides/getting-started/index.apt]: Good catch, done.

**Comment by mthmulders** [comment]
Your pull request has been merged into master. Thanks again for your contribution!

**Pull request closed without merging**